src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -23,9 +23,11 @@ public MessageReceiverNotifier(ICreateMessageReceivers receiverCreator, IConvert
         }
 
         public bool IsRunning { get; set; }
-        
+
         public int RefCount { get; set; }
-        
+
+        public CriticalError CriticalError { get; set; }
```

Comment:
Critical error isn't available in DI if I remember correctly?

Reply:
I think you are correct.